compiler/rustc_target/src/spec/windows_gnu_base.rs

```diff
@@ -16,34 +16,13 @@ pub fn opts() -> TargetOptions {
         ],
     );
 
-    let mut late_link_args = LinkArgs::new();
```

Comment:
I got this from https://github.com/msys2/MINGW-packages/blob/master/mingw-w64-rust/0007-clang-subsystem.patch, but am considering to create a separate `windows_gnu_base_clang.rs` to not interfere with the existing Windows GNU-based targets. Or do we first want to switch those targets over to Clang and then add `aarch64` when that's done? I came across some code that mentions:

> Note that we only do this on MSVC as I don't think
    clang has an output mode compatible with MinGW that we need. If it does we
    should switch to clang for MinGW as well!

https://github.com/rust-lang/rust/blob/0a4dc8bc161c68320a60a6bde525adae258b6252/src/ci/scripts/install-clang.sh#L31-L35

Reply:
I think we we'd need special triples for x86_64 and AArch64, `windows-gnu` aren't fully compatible with Clang based toolchains. That's why we have `clang*` subsystems at MSYS2.